java/code/src/com/suse/manager/webui/utils/gson/AnsiblePlaybookExecutionJson.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2021 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package com.suse.manager.webui.utils.gson;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.Date;
+import java.util.Optional;
+
+/**
+ * DTO for identifying ansible playbook execution
+ */
+public class AnsiblePlaybookExecutionJson {
```

Comment:
We need to support action chains with an optional parameter

Reply:
Done. Thanks for the reminder!